### ISSUE TITLE
Unsupported informat='multipart/form-data'

### ISSUE THREAD

**Issue by Katsivelisp** [freme-project/e-Publishing#21]
Hello,

invoking the e-publishing service at [http://api.freme-project.eu/current/e-publishing/html](http://api.freme-project.eu/current/e-publishing/html) returns the following error, even in the API Documentation page:
`{
  "exception": "eu.freme.common.exception.BadRequestException",
  "path": "/e-publishing/html",
  "message": "Unsupported informat='multipart/form-data'. The following serialization format values are acceptable: rdf-xml, application/xml, application/ld+json, n-triples, application/x-turtle, application/json, turtle, n3, application/rdf+xml, csv, text/xml, text/turtle, text/html, application/json+ld, xml, text/comma-separated-values, json, text, text/n3, application/x-openoffice, json-ld, application/n-triples, text/plain, application/x-xliff+xml",
  "error": "Bad Request",
  "status": 400,
  "timestamp": 1466584161257
}`
Running the same call on api-dev seems to be working. I have provided both valid metadata and a .zip file.

**Comment by pheyvaer** [comment]
So I think that this has something to do with the documentation page then, and not necessarily the service itself?

**Comment by Katsivelisp** [comment]
No, when I run the corresponding cURL command directly on http://api.freme-project.eu, I get the same error.

**Comment by pheyvaer** [comment]
but you said that 'Running the same call on api-dev seems to be working.'?

**Comment by Katsivelisp** [comment]
Yes, so:
[http://api.freme-project.eu/current/e-publishing/html](http://api.freme-project.eu/current/e-publishing/html) returns the error pasted above.
[http://api-dev.freme-project.eu/current/e-publishing/html](http://api-dev.freme-project.eu/current/e-publishing/html) works fine.

EDIT: You can use the cURL from #22 and switch the URL from dev to production.

**Comment by pheyvaer** [comment]
The e-Publishing service hasn't changed since the latest versions. @ArneBinder could you have a look at this? (Unless, this is something for @jnehring, but he is not available I thought)

**Comment by ArneBinder** [comment]
This is caused by a modification in the internalization filter. I adapted the changes in e-Publishing a month ago (registering supported mime types), so I'm not sure, why it was not released so far. When @jnehring does the next release of e-Publishing, this will be fixed in the live version.

**Comment by fsasaki** [comment]
Thanks, Arne. Is there a way for us to do the release or do we have to wait
until Jan is back?

2016-06-22 15:24 GMT+02:00 ArneBinder notifications@github.com:
